nettacker/probing/nmap_probes_converter.py

```
@@ -0,0 +1,364 @@
+                    probes.append(curr_probe)
+
+                line = line[len("Probe ") :]
+                protocol = line[:3]
+                line = line[4:]
+
+                name = line[0 : line.find(" ", 0)]
+
+                line = line[len(name) + 1 :]
+
+                if line[0] != "q":
+                    KeyError
```

Ruff (0.16.2)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)